src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueCreateDialog.kt

```diff
@@ -152,9 +152,11 @@ class IssueCreateDialog(
     private fun elementIsFocused(locator: By): ExpectedCondition<WebElement?> {
         return object : ExpectedCondition<WebElement?> {
             override fun apply(driver: WebDriver?): WebElement? {
-                val summary = driver!!.findElement(locator)
+                val element = driver!!.findElement(locator)
 
-                return if (summary == driver.switchTo().activeElement()) summary else null
+                driver.wait(elementToBeClickable(locator)).click()
+
+                return if (element == driver.switchTo().activeElement()) element else null
```

Comment:
If we don't want to wait in condition (within another wait), then we can delegate to `elementToBeClickable.apply` instead:
```kotlin
override fun apply(driver: WebDriver?): WebElement? {
    return elementToBeClickable(locator)
        .apply(driver)
        ?.also { it.click() }
        ?.takeIf { it == driver!!.switchTo().activeElement() }
}
```